packages/amazonq/test/unit/amazonqGumby/transformationResultsHandler.test.ts

```diff
@@ -10,11 +10,21 @@ import path from 'path'
 import { getTestResourceFilePath } from './amazonQGumbyUtil'
 import { fs } from 'aws-core-vscode/shared'
 
+type PatchDescription = {
+    name: string
+    fileName: string
+    isSuccessful: boolean
+}
+
 describe('DiffModel', function () {
     afterEach(() => {
         sinon.restore()
     })
 
+    const testDescription =
+        '{"name": "Added file", "fileName": "resources/files/addedFile.diffs", "isSuccessful": true}'
+    const parsedTestDescription: PatchDescription = JSON.parse(testDescription)
```

Comment:
Can put these 2 above in a `beforeEach` block above since we use both in both test cases